pkg/skaffold/util/util.go

```diff
@@ -89,6 +89,12 @@ func StrSliceContains(sl []string, s string) bool {
 func ExpandPathsGlob(workingDir string, paths []string) ([]string, error) {
 	expandedPaths := make(map[string]bool)
 	for _, p := range paths {
+		if _, err := os.Stat(p); err == nil {
```

Comment:
Can you instead use `AbsolutePaths` here?

Reply:
https://github.com/GoogleContainerTools/skaffold/blob/782c111a2ab1347a99e07cc8093f090f2b5517d8/pkg/skaffold/util/util.go#L265

Reply:
OK

Reply:
I use `filepath.IsAbs`, cause the `AbsolutePaths` is used to prepend the relative paths.